PlaceRouteHierFlow/placer/PlacerIfc.h

```diff
@@ -11,7 +11,7 @@ class PlacerIfc {
 
   public:
   PlacerIfc(PnRDB::hierNode& currentNode, int numLayout, string opath, int effort, PnRDB::Drc_info& drcInfo, const PlacerHyperparameters& hyper,
-            bool select_in_ILP);
+            bool select_in_ILP, int ilp_solver);
```

Comment:
The hyper parameters is probably a better place to put this (and also select_in_ILP), but this is no big deal.